fearless/Modules/Send/Validators/SendDataValidatingFactory.swift

```diff
@@ -104,20 +106,22 @@ class SendDataValidatingFactory: NSObject {
                     locale: locale
                 )
             }
-
             self?.basePresentable.presentExistentialDepositWarning(
                 existentianDepositValue: existentianDepositValue,
                 from: view,
                 action: {
                     delegate.didCompleteWarningHandling()
+                    warningHandler?()
                 },
                 locale: locale
             )
-
         }, preservesCondition: {
             guard !chainAsset.chain.isEthereum else {
                 return true
             }
+            if sendAllEnabled, canProceedIfViolated {
```

Comment:
looks wrong. should be sendAllEnabled || canProceedIfViolated

Reply:
no, if we enable full send AND chain can proceed violation - then we can do it, if anything isn't available - we can't